app/pods/publications/publication/controller.js

```diff
@@ -113,10 +113,30 @@ export default class PublicationController extends Controller {
     this.model.save();
   }
 
+  translationDateBeforePublication(fieldname, event) {
```

Comment:
Misschien hier hernoemen naar `isTranslationDateBeforePublicationDate` omdat het een boolean value teruggeeft?

Reply:
De logica zit nu in de twee aparte set methods n.a.v. opmerking sven.